### DISCUSSION TITLE
Thank You!

### DISCUSSION THREAD

**Discussion by TyTyThei** [Godiesc/firefox-gx#82] [General]
Dear Contributors of _Firefox-Gx_,

Thank You!

I just wanted to say thank you guys for making this amazing theme.

Thank you for fixing bugs.
Thank you for quickly getting out bug fixes.
Just thank you for the (probably) hard work you put into this.

Thank You!

Sincerely,

Tyrichyrich

**Comment by keldian** [comment]
Ditto. Keep up the good work.

**Comment by Ser4ph4** [comment]
Thank you very much for all the time you've dedicated to this, you're always very helpful in helping people here, keep going my friend.
[Godiesc](https://github.com/Godiesc) 🤗
